cmd/ssh-tpm-agent/main.go

```diff
@@ -201,12 +205,21 @@ func main() {
 			return tpm
 		},
 
+		// Owner password
+		func() ([]byte, error) {
+			if askOwnerPassword {
+				return pinentry.GetOwnerPassword()
+			} else {
+				return []byte(""), nil
+			}
+		},
```

Comment:
Should we have a environment variable for ownerpass as well?

Reply:
I am not a big fan of passing secrets via environment variables but there may be cases where this is desirable.